# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../plptools/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- plptools-1.0.4/ncpd/channel.cc.vanilla	2008-06-12 13:27:56.000000000 +0200
+++ plptools-1.0.4/ncpd/channel.cc	2008-06-12 13:28:03.000000000 +0200
@@ -28,6 +28,8 @@
 #include <iostream>
 #include <string>
 
+#include <string.h>
+
 #include "channel.h"
 #include "ncp.h"
 
--- plptools-1.0.4/ncpd/packet.cc.vanilla	2008-06-12 13:29:07.000000000 +0200
+++ plptools-1.0.4/ncpd/packet.cc	2008-06-12 13:29:15.000000000 +0200
@@ -29,6 +29,7 @@
 #include <fstream>
 #include <iomanip>
 
+#include <string.h>
 #include <stdio.h>
 #include <malloc.h>
 #include <unistd.h>
--- plptools-1.0.4/ncpd/linkchan.cc.vanilla	2008-06-12 13:28:15.000000000 +0200
+++ plptools-1.0.4/ncpd/linkchan.cc	2008-06-12 13:28:30.000000000 +0200
@@ -24,6 +24,8 @@
 #include <iomanip>
 #include <string>
 
+#include <string.h>
+
 #include <bufferstore.h>
 #include <bufferarray.h>
 
--- plptools-1.0.4/ncpd/main.cc.vanilla	2008-06-12 13:28:38.000000000 +0200
+++ plptools-1.0.4/ncpd/main.cc	2008-06-12 13:28:44.000000000 +0200
@@ -28,6 +28,8 @@
 #include <string>
 #include <iostream>
 
+#include <string.h>
+
 #include <bufferstore.h>
 #include <ppsocket.h>
 #include <iowatch.h>
--- plptools-1.0.4/lib/ppsocket.cc.vanilla	2008-06-12 13:26:30.000000000 +0200
+++ plptools-1.0.4/lib/ppsocket.cc	2008-06-12 13:26:38.000000000 +0200
@@ -31,6 +31,7 @@
 
 #include <iostream>
 
+#include <string.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <stdarg.h>
--- plptools-1.0.4/lib/Enum.cc.vanilla	2008-06-12 13:27:27.000000000 +0200
+++ plptools-1.0.4/lib/Enum.cc	2008-06-12 13:27:33.000000000 +0200
@@ -24,6 +24,8 @@
 #include <config.h>
 #endif
 
+#include <string.h>
+
 #include "Enum.h"
 
 using namespace std;
--- plptools-1.0.4/lib/bufferstore.cc.vanilla	2008-06-12 13:25:55.000000000 +0200
+++ plptools-1.0.4/lib/bufferstore.cc	2008-06-12 13:26:19.000000000 +0200
@@ -28,10 +28,10 @@
 
 #include "bufferstore.h"
 
-// Should be iostream.h, but won't build on Sun WorkShop C++ 5.0
 #include <iomanip>
 #include <string>
 
+#include <string.h>
 #include <stdlib.h>
 #include <ctype.h>
 #include <assert.h>
--- plptools-1.0.4/lib/plpdirent.h.vanilla	2008-06-12 13:26:55.000000000 +0200
+++ plptools-1.0.4/lib/plpdirent.h	2008-06-12 13:27:05.000000000 +0200
@@ -26,6 +26,8 @@
 #include <iostream>
 #include <string>
 
+#include <string.h>
+
 #include <psitime.h>
 #include <rfsv.h>
 
